css-brochure-form/styles.css

```diff
@@ -0,0 +1,57 @@
+* {
+  font-family: 'Brandon Grotesque', sans-serif;
+}
+
+h2 {
+  text-transform: uppercase;
+  color: #ec5c57;
+  justify-content: center;
```

Comment:
If the `h2` doesn't have `display: flex`, this `justify-content` isn't doing anything